### DISCUSSION TITLE
Improvements for Cursor Size based on map's Circle Size

### DISCUSSION THREAD

**Discussion by Totsugeki1** [ppy/osu#15166] [Ideas] [answered]
First time using GitHub, please excuse the mistakes.

I think the user should be able to do a lot more with cursor size, and adjust it according to the CS he is playing, not just a multiplier of the value of the cursor size. What do I mean by this? It would be great if the player was able to select the size of the cursor he wants for different CS sizes. This could easily be done by just letting users set a cursor size for the slower circle size in the game and the biggest one, but I think adding midpoints would make this a lot better (this would allow for users to, for example, make their cursor stop growing in size after certain CS or to make the change different depending on the CS range). For example, the cursor size change is proportional from CS 6 to 8 as CS 4 to 6, but maybe the player wants the change in cursor size between 4 and 6 to be higher in comparison to 6 to 8. The calculations would just be made using the values the user gave for the limits of that range of CS (range would be, for example, CS 1 to 5). An example for this would be (obviously it wouldn't be done this way in-game): "CS 0-3 1.2x, CS 3 1.0x, CS 5 0.75x, CS 8 0.5x, CS 10 0.45x".

**Comment by peppy** [comment, marked as answer]
out of scope. maybe you can use settings profiles for this when we have them.
